fdbserver/DDTeamCollection.actor.cpp

```diff
@@ -2417,12 +2404,13 @@ class DDTeamCollectionImpl {
 				if (self->configuration.perpetualStorageWiggleLocality == "0") {
 					isr.storeType = self->configuration.perpetualStoreType;
 				} else {
-					Optional<Value> localityKey;
-					Optional<Value> localityValue;
-					ParsePerpetualStorageWiggleLocality(
-					    self->configuration.perpetualStorageWiggleLocality, &localityKey, &localityValue);
-					if (candidateWorker.worker.locality.get(localityKey.get()) == localityValue) {
-						isr.storeType = self->configuration.perpetualStoreType;
+					std::vector<std::pair<Optional<Value>, Optional<Value>>> localityKeyValues =
+					    ParsePerpetualStorageWiggleLocality(self->configuration.perpetualStorageWiggleLocality);
+					for (const auto& [localityKey, localityValue] : localityKeyValues) {
```

Comment:
Is it possible to use `localityMatchInList` here?

Reply:
Good point. I probably wrote this part first before I created `localityMatchInList`